README.md

```diff
@@ -29,3 +29,7 @@ gradlew build
 ```bash
 gradlew run
 ```
+
+##DevOps Documentation
+Our documentation resides into the Confluence space.
+Below is the link: https://stefanobraggion.atlassian.net/l/c/qVfgiRaV
```

Comment:
I think the following is better:

`You can read it on the [DevOps page](https://stefanobraggion.atlassian.net/l/c/qVfgiRaV "Confluence DevOps page")`